.packit.yaml

```diff
@@ -0,0 +1,21 @@
+---
+specfile_path: python3.spec
+actions:
+  post-upstream-clone:
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/python3.spec
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/idle3.appdata.xml
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/idle3.desktop
+  - curl -O https://src.fedoraproject.org/rpms/python3/raw/master/f/check-pyc-timestamps.py
+  # packit will apply the patches itself
+  - sed '/^Patch/d' -i python3.spec
+  # patckit uses %autosetup - and yes, the line below doesn't make sense given
+  # how python3's spec look, this is just a demonstration of packit's functionality
+  - sed '/^%patch/d' -i python3.spec
+# this commit should match the upstream tarball, all the commits on top are treated as downstream patches
+upstream_ref: 802eb676bad7a2558fdf35a8ea5882f5e97c2ad8
```

Comment:
Why do I need to manually track this?

Reply:
at this moment, packit is not smart enough to detect a commit matches a release tarball

how would you expect it to work? (the easiest solution for me would be to pick the latest tagged commit)

Reply:
> the easiest solution for me would be to pick the latest tagged commit

Yes, that'll do. We also tag our releases here with `fedora-<version>`, so a match-tag regex/glob argument would be needed.

Reply:
oh yeah, that would be smart - that way you wouldn't need to touch that field, ever

https://github.com/packit-service/packit/issues/823

Reply:
You can use any git-ref, so tag and branch is valid as well. But search for last tag sounds good.